include/mpm_base.tcc

```diff
@@ -48,6 +48,18 @@ mpm::MPMBase<Tdim>::MPMBase(const std::shared_ptr<IO>& io) : mpm::MPM(io) {
       velocity_update_ = false;
     }
 
+    // Damping
```

Comment:
Move this to a separate function, which returns a boolean. Boolean is set to false, when an exception is caught